packages/admin/admin/src/mui/menu/ItemGroup.tsx

```diff
@@ -21,10 +22,12 @@ const styles = (theme: Theme) =>
             color: `${theme.palette.grey[300]}`,
         },
         titleMenuOpen: {
+            fontWeight: 600,
             fontSize: 14,
             border: `2px solid ${theme.palette.common.white}`,
             borderRadius: "initial",
             padding: 0,
+            marginRight: theme.spacing(1),
```

Comment:
Shouldn't this be part of the helper icon?

Reply:
imo passing an icon without styles should result in an acceptable outcome. And since the margin doesn't hurt on the title I'd keep it

Reply:
You could add the margin to the icon if one exists. Should work like this:

```
titleMenuOpen: {
    [`& + .${svgIconClasses.root}`]: {
        marginLeft: theme.spacing(1),
    },
}
```

But that's a minor detail IMO

Reply:
We can keep it like it currently is.